tests/EventListener/FileListenerTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Tests\AppBundle\FileStorage;
+
+use AppBundle\Entity\Mooc\AttachmentFile;
+use AppBundle\EntityListener\FileListener;
+use League\Flysystem\Filesystem;
+use League\Glide\Server;
+use Liip\FunctionalTestBundle\Test\WebTestCase;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+
+class FileListenerTest extends WebTestCase
+{
+    private $entityFileListener;
+
+    public function testProcessFile(): void
+    {
+        $entityFile = new AttachmentFile();
+        $entityFile->setFile(new UploadedFile(__FILE__, __FILE__));
+
+        $this->entityFileListener->processFile($entityFile);
+
+        self::assertNotNull($entityFile->getPath());
+        self::assertNull($entityFile->getFile());
+    }
+
+    protected function setUp()
+    {
+        self::bootKernel();
+        $this->containers = self::$kernel->getContainer();
```

Comment:
`containers` is supposed to be an array of containers (see https://github.com/liip/LiipFunctionalTestBundle/blob/master/Test/WebTestCase.php#L260)

Can't you use `$container = $this->getContainer()` here instead ?